### DISCUSSION TITLE
New Arabic words and quotes?

### DISCUSSION THREAD

**Discussion by zefr0x** [monkeytypegame/monkeytype#1423] [Questions] [answered]
I'm currently working on arranging, fixing and filtering a list of 10,000 Arabic most used words, and I wanted to ask about several things before I starte creating other lists after thet one, because i don't know if all of my listing ideas coincide with the project vision.

I have a plan to make lists and they are in order of priority as follows:
- [ ] **Arabic 10k** (currently working on)
- [ ] **Arabic Qur’an words**
1. The [Qur’an](https://en.wikipedia.org/wiki/Quran) has a deep link with the Arabic language, the Qur’an added many meanings and vocabulary to the language, and after its revelation the Arabic language became an educational language with disciplined rules, the Qur’an is considered the largest pillar of the Arabic language and is referred to in grammar.
2. It contains many words, descriptions, and meanings of all kinds of words
3. Many courts in Arab countries refer to the Qur’an in their rulings, so the list will be useful to court writers.

(It is a religious text, the words are many and have many meanings, I want to add them due to its great connection to the language, and I do not mean by that any religious prejudice)

- [ ] **Arabic names**
1. This list will be useful for reception employees, whether in hotels or hospitals etc...
2. It may be useful for government departments and banks where names are frequently written.

-------------
I also want to add many quotes so can i make a pull request on github? Because I read that the it has some thing to do with the discord server and i don't use discord.

For **"Arabic 10k"** and  **"Arabic Qur’an words"** I will create two versions, one with the [diacritics](https://en.wikipedia.org/wiki/Arabic_diacritics) and the other without because some people may not use the diacritics, but it is an essential part of the language and almost all the words have different meanings by changing the only one diacritics without changing anything in the letters. Many people may not use them because the words may be easy to guess without diacritics. However, many sensitive texts, which may have complex or archaic signs, are written with full diacritics.

**Comment by Miodec** [comment, marked as answer]
I think hold off on creating separate diacritic free version as there might be a separate setting so it works for all languages (all though I'm still considering the implications of it).
